### PR TITLE
improve codecov for clp/types

### PR DESCRIPTION
**Pull request by wilsonc2020**
* improve codecov for clp/types.
* fix bugs around using `GTE`.
* Remove unused methods.

### CODE REVIEW

File: x/clp/types/params.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ func (p Params) Validate() error {
 	return validateMinCreatePoolThreshold(p.MinCreatePoolThreshold)
 }
 
+//(skip thi one because this method invoked by validate)
```
Comment (timlind) [inline x/clp/types/params.go:51]: can remove this comment

File: x/clp/types/params_test.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_ParamKeyTable(t *testing.T) {
+	_ = ParamKeyTable()
```
Comment (timlind) [inline x/clp/types/params_test.go:10]: Can be removed unless it is going to be filled in with a meaningful test. The same is true in other places.

File: x/clp/types/querier.go [hunk 3]
```diff
@@ -17,15 +13,3 @@ const (
 func NewQueryReqGetPool(symbol string) PoolReq {
 	return PoolReq{Symbol: symbol}
 }
-
```
Comment (timlind) [inline x/clp/types/querier.go:20]: Why are these being removed?

File: x/clp/types/msgs.go [hunk 4]
```diff
@@ -72,8 +72,8 @@ func (m MsgSwap) ValidateBasic() error {
 	if !m.ReceivedAsset.Validate() {
 		return sdkerrors.Wrap(ErrInValidAsset, m.ReceivedAsset.Symbol)
 	}
-
-	if m.SentAsset == m.ReceivedAsset {
+	// == compare the address instead of value here so I think we should use .Equals here.
```
Comment (timlind) [inline x/clp/types/msgs.go:75]: Agreed, this comment can be removed.

File: x/clp/types/genesis_test.go [hunk 5]
```diff
@@ -0,0 +1,21 @@
+package types
+
+import (
+	//"fmt"
+
+	// "encoding/hex"
```
Comment (timlind) [inline x/clp/types/genesis_test.go:6]: commented out code can be removed.

File: x/clp/types/msgs.go [hunk 6]
```diff
@@ -156,7 +156,7 @@ func (m MsgAddLiquidity) ValidateBasic() error {
 	if m.ExternalAsset.Equals(GetSettlementAsset()) {
 		return sdkerrors.Wrap(ErrInValidAsset, "External asset cannot be rowan")
 	}
-	if !(m.NativeAssetAmount.GTE(sdk.ZeroUint())) && (m.ExternalAssetAmount.GTE(sdk.ZeroUint())) {
+	if (!m.NativeAssetAmount.GT(sdk.ZeroUint())) && (!m.ExternalAssetAmount.GT(sdk.ZeroUint())) {
```
Comment (timlind) [inline x/clp/types/msgs.go:159]: 👍🏻

File: x/clp/types/querier.go (hunk 3 above)
Comment (wilsonc2020) [reply x/clp/types/querier.go:20]: Synced up with Tim offline, so we are good with remove these unused methods.

**Review by khdegraaf — APPROVED** [review]
I fixed some minor remaining formatting cleanup issues.

**Pull request merged**